2022-01-21 15:11:17,436 INFO    MainThread:2730 [wandb_setup.py:_flush():71] setting env: {}
2022-01-21 15:11:17,436 INFO    MainThread:2730 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-21 15:11:17,436 INFO    MainThread:2730 [wandb_init.py:_log_setup():371] Logging user logs to /Users/gdegournay/PycharmProjects/Udacity/MLOps/MLOps_ex/reproducible workflow/wandb/run-20220121_151117-3hxmbgrw/logs/debug.log
2022-01-21 15:11:17,437 INFO    MainThread:2730 [wandb_init.py:_log_setup():372] Logging internal logs to /Users/gdegournay/PycharmProjects/Udacity/MLOps/MLOps_ex/reproducible workflow/wandb/run-20220121_151117-3hxmbgrw/logs/debug-internal.log
2022-01-21 15:11:17,437 INFO    MainThread:2730 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fcd98230a30>
2022-01-21 15:11:17,437 INFO    MainThread:2730 [wandb_init.py:init():404] calling init triggers
2022-01-21 15:11:17,437 INFO    MainThread:2730 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-21 15:11:17,437 INFO    MainThread:2730 [wandb_init.py:init():460] starting backend
2022-01-21 15:11:17,437 INFO    MainThread:2730 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-01-21 15:11:17,440 INFO    MainThread:2730 [backend.py:ensure_launched():216] starting backend process...
2022-01-21 15:11:17,444 INFO    MainThread:2730 [backend.py:ensure_launched():221] started backend process with pid: 2763
2022-01-21 15:11:17,448 INFO    MainThread:2730 [wandb_init.py:init():469] backend started and connected
2022-01-21 15:11:17,462 INFO    MainThread:2730 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-01-21 15:11:17,462 INFO    MainThread:2730 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-01-21 15:11:17,462 INFO    MainThread:2730 [wandb_init.py:init():533] updated telemetry
2022-01-21 15:11:17,465 INFO    MainThread:2730 [wandb_init.py:init():563] communicating current version
2022-01-21 15:11:18,344 INFO    MainThread:2730 [wandb_init.py:init():568] got version response 
2022-01-21 15:11:18,344 INFO    MainThread:2730 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-21 15:11:18,674 INFO    MainThread:2730 [wandb_init.py:init():606] starting run threads in backend
2022-01-21 15:11:20,268 INFO    MainThread:2730 [wandb_run.py:_console_start():1810] atexit reg
2022-01-21 15:11:20,269 INFO    MainThread:2730 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-01-21 15:11:20,269 INFO    MainThread:2730 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-01-21 15:11:20,270 INFO    MainThread:2730 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-21 15:11:20,270 INFO    MainThread:2730 [wandb_init.py:init():633] run started, returning control to user process
2022-01-21 15:11:20,271 INFO    MainThread:2730 [wandb_init.py:_pause_backend():284] pausing backend
2022-01-21 15:11:20,271 INFO    MainThread:2730 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-01-21 15:11:58,662 INFO    MainThread:2730 [wandb_init.py:_resume_backend():293] resuming backend
2022-01-21 15:11:58,665 INFO    MainThread:2730 [wandb_init.py:_pause_backend():284] pausing backend
2022-01-21 15:11:58,665 INFO    MainThread:2730 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-01-21 15:12:02,357 INFO    MainThread:2730 [wandb_init.py:_resume_backend():293] resuming backend
2022-01-21 15:12:02,718 INFO    MainThread:2730 [wandb_init.py:_pause_backend():284] pausing backend
2022-01-21 15:12:02,718 INFO    MainThread:2730 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-01-21 15:12:15,670 INFO    MainThread:2730 [wandb_init.py:_resume_backend():293] resuming backend
2022-01-21 15:12:15,670 INFO    MainThread:2730 [wandb_run.py:finish():1457] finishing run geodego/demo_artifact/3hxmbgrw
2022-01-21 15:12:15,670 INFO    MainThread:2730 [jupyter.py:save_history():429] not saving jupyter history
2022-01-21 15:12:15,671 INFO    MainThread:2730 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-01-21 15:12:15,671 INFO    MainThread:2730 [wandb_init.py:_jupyter_teardown():303] cleaning up jupyter logic
2022-01-21 15:12:15,671 INFO    MainThread:2730 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-01-21 15:12:15,671 INFO    MainThread:2730 [wandb_run.py:_restore():1752] restore
2022-01-21 15:12:16,064 INFO    MainThread:2730 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 662
  total_bytes: 662
}

2022-01-21 15:12:16,378 INFO    MainThread:2730 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 662
  total_bytes: 662
}

2022-01-21 15:12:16,869 INFO    MainThread:2730 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 662
  total_bytes: 3383
}

2022-01-21 15:12:16,970 INFO    MainThread:2730 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 662
  total_bytes: 10115
}

2022-01-21 15:12:17,077 INFO    MainThread:2730 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 662
  total_bytes: 10115
}

2022-01-21 15:12:17,183 INFO    MainThread:2730 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 662
  total_bytes: 10115
}

2022-01-21 15:12:17,286 INFO    MainThread:2730 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 662
  total_bytes: 10115
}

2022-01-21 15:12:17,388 INFO    MainThread:2730 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 10115
  total_bytes: 10115
}

2022-01-21 15:12:17,493 INFO    MainThread:2730 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 10115
  total_bytes: 10115
}

2022-01-21 15:12:17,595 INFO    MainThread:2730 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 10115
  total_bytes: 10115
}

2022-01-21 15:12:17,700 INFO    MainThread:2730 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 10115
  total_bytes: 10115
}

2022-01-21 15:12:17,807 INFO    MainThread:2730 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 10115
  total_bytes: 10115
}

2022-01-21 15:12:17,913 INFO    MainThread:2730 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 10115
  total_bytes: 10115
}

2022-01-21 15:12:18,621 INFO    MainThread:2730 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 10115
  total_bytes: 10115
}

2022-01-21 15:12:19,037 INFO    MainThread:2730 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 10115
  total_bytes: 10115
}
local_info {
}

2022-01-21 15:12:20,157 INFO    MainThread:2730 [wandb_run.py:_append_files():2180] logging synced files
